ai-machine-learning/prometheus-timeseries-forecasting/finish.md

```diff
@@ -1,3 +1,31 @@
-The contents of this scenario are available on [github](https://github.com/hemajv/prometheus-anomaly-detection-workshop)
+## Feedback for this scenario
 
-Also check out other Red Hat projects [radanalytics.io](https://radanalytics.io/) and [Open Data Hub](https://opendatahub.io/).
+Did you like this scenario? or Do you think we can improve on something? <br>
+You can help us by providing your feedback/suggestions [here](https://docs.google.com/forms/d/e/1FAIpQLSe9BgNpYx9KrmyAOE4UiJmEw04Mcff6Ob8HHVFxBygDdGCAEw/viewform?entry.770234109=Data+Science+with+Prometheus+Metrics).
+
+## Continue Learning
+
+### More information about the applications we used
+
+* [**Prometheus**](https://github.com/prometheus/prometheus#prometheus) - A Cloud native application monitoring system.
+* [**Jupyter Notebook**](https://jupyter.org/) - Open source web application for creating and sharing documents that contain live code, equations, visualizations and text.
+
```

Comment:
Can you also add a section to showcase the Prometheus Anomaly Detector?

Reply:
added :+1: